staging/src/k8s.io/client-go/rest/watch/decoder_test.go

```diff
@@ -93,6 +95,7 @@ func TestDecoder(t *testing.T) {
 		}()
 		<-done
 
+		close(errs)
```

Comment:
this is similar to the other place we discussed before. We cannot close the channel without calling `t.Fatalf`. Better be fixed in a separate PR.

Reply:
got it.